### PR TITLE
feat: added Ad and Email classes

### PR DESCRIPTION
**Pull request by Trustacean**

### CODE REVIEW

**Comment by vianneynara** [comment]
kode lu jelek

**Comment by Trustacean** [comment]
balls

File: src/main/java/models/Account.java [hunk 1]
```diff
@@ -14,50 +14,61 @@
  * */
 
 public class Account {
-
+	// Attribute declarations
```
Comment (vianneynara) [inline src/main/java/models/Account.java:17]: Please remove this

Comment (Trustacean) [reply src/main/java/models/Account.java:17]: balls

Comment (vianneynara) [inline src/main/java/models/Account.java:17]: Please remove this

**Review by vianneynara — CHANGES_REQUESTED** [review]
Please delete this

File: src/main/java/models/Account.java (hunk 1 above)
Comment (Trustacean) [reply src/main/java/models/Account.java:17]: Kys

**Pull request merged**